Assets/Scripts/GrapplingArrow.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GrapplingArrow : MonoBehaviour
+{
+    PlayerManager player = PlayerManager.instance;
+    public float moveSpeed = 1;
+    public int massThreshold = 4;
+    Rigidbody arrowRB;
+    // Start is called before the first frame update
+    void Start()
+    {
+        arrowRB = GetComponent<Rigidbody>();
+    }
+    void OnCollisionEnter(Collision collision)
+    {
+        //These are the varous variables that will be used
+        var player = PlayerManager.instance;
```

Comment:
Are these variables being used? They dont seem to be doing anything

Reply:
Good catch. I removed the variables in my most recent commit.